games-roguelike/tome/tome-2.3.5.ebuild

```diff
@@ -28,14 +28,19 @@ DEPEND="${RDEPEND}
 
 S=${WORKDIR}/tome-${MY_PV}-src/src
 
+PATCHES=( "${FILESDIR}/${PN}-2.3.5-gentoo-paths.patch"
+	"${FILESDIR}"/${PN}-2.3.5-format.patch
+	"${FILESDIR}"/${PN}-2.3.5-noX.patch
+)
+
 src_prepare() {
 	mv makefile.std makefile
-	epatch "${FILESDIR}/${PV}-gentoo-paths.patch" \
-		"${FILESDIR}"/${P}-format.patch \
-		"${FILESDIR}"/${P}-noX.patch
+
+	default
+
 	sed -i -e '/^CC =/d' makefile || die
 	sed -i -e "s:xx:x:" ../lib/edit/p_info.txt || die
-	sed -i -e "s:GENTOO_DIR:${GAMES_STATEDIR}:" files.c init2.c || die
+	# sed -i -e "s:GENTOO_DIR:${GAMES_STATEDIR}:" files.c init2.c || die
 
 	find .. -name .cvsignore -exec rm -f \{\} +
 	find ../lib/edit -type f -exec chmod a-x \{\} +
```

Comment:
Below, there's a direct pkg-config call, but we want to use the helper function from toolchain.eclass.